docs/how-to/performance_guidelines.rst

```diff
@@ -275,7 +259,7 @@ allows for more fine-grained control over the execution order of commands,
 which can be beneficial in certain scenarios.
```

Comment:
Replace lines 256 to 259 with:

An alternative way to synchronize is to use streams. Different streams can execute commands either without following a specific order or concurrently. This is why streams allow more fine-grained control over the execution order of commands, which can be beneficial in certain scenarios.

Reply:
Fixed.